### PR TITLE
Buffs the Bulldog

### PR DESCRIPTION
**Pull request by Borisvanmemes**
## About The Pull Request

Buffs the Bulldog via giving it a damage modifier of 1.2.

## Why It's Good For The Game

Assuming default ammo, the C-20r is 30 damage a shot. The M-90gl is 35 damage, and has grenades as a side dish. The Saw is 30 and is fully automatic. The Sniper is 70 alongside having incredible range, stunning, and delimbing. 

The Bulldog when loaded with buckshot does a grand total of 45 damage point blank. 

That's a pretty rough comparison, and once you take into account the other types of ammo nukie guns can use, it doesn't get much better. As is the only real usecase for the Bulldog is meteor slugs, and you're not really using those for murder anyways.

## Changelog

:cl:
balance: Bulldogs now hit 20% harder with their projectiles!
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 1]
```diff
@@ -157,6 +158,7 @@
 	var/secondary_magazine_type
 	///the secondary magazine
 	var/obj/item/ammo_box/magazine/secondary_magazine
+	projectile_damage_multiplier = 1.2
```
Comment (san7890) [inline code/modules/projectiles/guns/ballistic/shotgun.dm:161]: move this above all of the new variable declarations

**Review by Fikou — CHANGES_REQUESTED** [review]
you dont give any explanation why making it normal sized should be a thing. the bulldog isn't a sidearm like the pistol, i disagree that it should be normal sized (the c20r and sniper shouldnt be either but thats for a different pr)

File: code/modules/projectiles/guns/ballistic/shotgun.dm (hunk 1 above)
Comment (Borisvanmemes) [reply code/modules/projectiles/guns/ballistic/shotgun.dm:161]: I'm going to blame me being unable to understand this on it being 2:30AM but by new variable declarations you mean everything below the inhand dimensions vars?

**Comment by Borisvanmemes** [comment]
> you dont give any explanation why making it normal sized should be a thing. the bulldog isn't a sidearm like the pistol, i disagree that it should be normal sized (the c20r and sniper shouldnt be either but thats for a different pr)

After doing some rethinking, you're probably right. I can't find any good reason as to why it should be normal sized apart from nostalgia.

File: code/modules/projectiles/guns/ballistic/shotgun.dm (hunk 1 above)
Comment (Borisvanmemes) [reply code/modules/projectiles/guns/ballistic/shotgun.dm]: Uh, it should be good now?

**Review by Fikou — APPROVED** [review]

**Comment by SomeoneYouProbablyKnow** [comment]
All projectiles? Slugs included? Scary

**Pull request merged**